setup/paths.sh

```diff
@@ -37,38 +37,27 @@ else
     source $BUILD_HOME/cmsgemos/setup/paths.sh
 fi
 
+# Export project
+export GEM_PLOTTING_PROJECT=$BUILD_HOME/gem-plotting-tools
+
+# Setup Path
+export PATH=$PATH:$GEM_PLOTTING_PROJECT
+export PATH=$PATH:$GEM_PLOTTING_PROJECT/macros
+export PATH=$PATH:$GEM_PLOTTING_PROJECT/setup
+
+# Setup PYTHONPATH
+export PYTHONPATH=$PYTHONPATH:$BUILD_HOME/vfatqc-python-scripts
+export PYTHONPATH=$PYTHONPATH:$GEM_PLOTTING_PROJECT
+export PYTHONPATH=$PYTHONPATH:$GEM_PLOTTING_PROJECT/macros
+export PYTHONPATH=$PYTHONPATH:$GEM_PLOTTING_PROJECT/setup
```

Comment:
contents of macros and setup are used in the various python scripts here.

e.g. `fitScanData.py` (in $GEM_PLOTTING_PROJECT/macros) is needed by anaUltra*.py routines and channel map info (in $GEM_PLOTTING_PROJECT/setup) is needed by anaUltra*.py